lib/gooddata/core/connection.rb

```diff
@@ -0,0 +1,376 @@
+# encoding: UTF-8
+
+require 'multi_json'
+require 'rest-client'
+
+require_relative '../version'
+
+module GoodData
+  # # GoodData HTTP wrapper
+  #
+  # Provides a convenient HTTP wrapper for talking with the GoodData API.
+  #
+  # Remember that the connection is shared amongst the entire application.
+  # Therefore you can't be logged in to more than _one_ GoodData account.
+  # per session. Simultaneous connections to multiple GoodData accounts is not
+  # supported at this time.
```

Comment:
@fluke777, to s tim ze nemuzes byt pripojen do vice nez jednoho uctu neni uplne pravda, ne?